src/components/shared/ArrowPanel.css

```diff
@@ -12,7 +12,7 @@
 .arrowPanel {
   --internal-offset-from-right: var(--offset-from-right, 60px);
   --internal-offset-from-top: 15px;
-  --internal-width: var(--width, auto);
+  --internal-width: var(--width, initial);
```

Comment:
`min-width` doesn't know the `auto` keyword. I used `initial` instead which is valid for all properties.